spec/cassettes/Telegram_Bot_Api/_call/has_result.yml

```diff
@@ -0,0 +1,42 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://api.telegram.org/bot<BOT_API_TOKEN>/test/getMe
+    body:
+      encoding: UTF-8
+      string: ''
+    headers:
+      User-Agent:
+      - Faraday v2.7.10
+      Content-Type:
+      - application/x-www-form-urlencoded
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      server:
+      - nginx/1.18.0
+      date:
+      - Mon, 17 Jul 2023 09:06:03 GMT
+      content-type:
+      - application/json
+      content-length:
+      - '200'
+      connection:
+      - keep-alive
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains; preload
+      access-control-allow-origin:
+      - "*"
+      access-control-allow-methods:
+      - GET, POST, OPTIONS
+      access-control-expose-headers:
+      - Content-Length,Content-Type,Date,Server,Connection
+    body:
+      encoding: UTF-8
+      string: '{"ok":true,"result":{"id":2200529744,"is_bot":true,"first_name":"Test
+        bot","username":"alexwayfer_test_bot","can_join_groups":true,"can_read_all_group_messages":false,"supports_inline_queries":false}}'
```

Comment:
I can mask some data here if it confuses you, but I see nothing confidential here, and see no problems with different data for different cassettes (if somebody else will write new specs or re-record existing ones).

Reply:
Well yeah, nothing sensitive here. Let's leave it like this for now.